### PR TITLE
Marwa-exercises- week1

### PR DESCRIPTION
**Pull request by Marwa-kwara**

### CODE REVIEW

File: Week1/Homework/exercise6.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+console.log("I think arry is empty and the out put of it is undefind");
+console.log(zooAnimals);
+
+let myFavoriteAnimals =[
+    "cats",{favoritePets:"rabbits",iLoveDogs:"false",}
+]
+console.log(typeof myFavoriteAnimals);
+console.log(myFavoriteAnimals);
+
+myFavoriteAnimals.onMoreAnimal=["piglet"];
```
Comment (jasperkennis) [inline Week1/Homework/exercise6.js:11]: I think you are mixing **objects** and **arrays** here. Note that the outcome of your script is this:

```
I think arry is empty and the out put of it is undefind
[]
object
[ 'cats', { favoritePets: 'rabbits', iLoveDogs: 'false' } ]
[ 'piglet' ]
```

Insteed of `myFavoriteAnimals.onMoreAnimal=["piglet"];`, try using `myFavoriteAnimals.push('piglet')` and see how the result is different.

In the end, we should first console.log 4 animals, and then 5.

File: Week1/Homework/exercise9.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+let x=7;
```
Comment (jasperkennis) [inline Week1/Homework/exercise9.js:1]: I can see that the values of your variables are correct. But let's practise something else: for completeness sake, try to assign `x = x % 3`, and then console.log the **new** value of x.

File: Week1/Homework/exercise10.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+let seasons=[
+    'winter','spring','summer','autumn'
+]
+let monthes=['january', 'february', 'march', 'april', 'may', 'june', 'july',]
+console.log(seasons.length);
+console.log(monthes.length);
+if (seasons > monthes);
+console.log('Two different sizes');
```
Comment (jasperkennis) [inline Week1/Homework/exercise10.js:8]: The first part here is correct, but the if statement isn't entirely complete yet. Please see if you can find a good example of an if statement. If you get stuck on this, just let me know and we can look at it together.

**Review by jasperkennis — CHANGES_REQUESTED** [review]
Hi Marwa! 

Nice work with the homework! There are a few concepts I think would be good to check a bit more, please see my comments. Let me know if you have any questions!

Enjoy the weekend!

File: Week1/Homework/exercise6.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+let zooAnimals=[]
+console.log("I think arry is empty and the out put of it is undefind");
+console.log(zooAnimals);
```
[the remark this answers is not in the document]
Comment (Marwa-kwara) [reply Week1/Homework/exercise6.js:3]: okay !

**Comment by Marwa-kwara** [comment]
Hi @jasperkennis,
Thank you for reviewing my homework,I did some changes on exercises 6/9/10, can you please check it again?
Thanks in advance,

**Comment by Marwa-kwara** [comment]
some changes on the exercises 6,9,10

File: Week1/Homework/exercise10.js (hunk 3 above)
Comment (jasperkennis) [reply Week1/Homework/exercise10.js]: Hi Marwa! Nice work with the updates. There's one important thing left, about this if statement.

You have this: 
```js
if (seasons.length > monthes.length);
console.log('Two different sizes');
```

Actually, the if statement should be something like this:
```js
if (seasons.length > monthes.length) {
  console.log('Two different sizes');
} else {
  console.log('The same sizes');
}
```

Notice the curly brackets that indicate the block of logic that happens _if_ the condition is met.

Comment (Marwa-kwara) [reply Week1/Homework/exercise10.js]: Oh yes, I forgot to add the curly brackets,
Thank you ,

Comment (jasperkennis) [reply Week1/Homework/exercise10.js]: I think you might not have pushed the changes, but I can see from your homework for week2 that this is now clear to you:)

**Pull request closed without merging**